supfile.go

```diff
@@ -13,21 +13,66 @@ import (
 	"gopkg.in/yaml.v2"
 )
 
+// EnvVar represents an environment variable
+type EnvVar struct {
+	Key   string
+	Value string
+}
```

Comment:
Correct, this is in my mind for a long run. What do you think about it?

But it needs more thinking, as we want to implement `onerror` #42 and `if` conditionals #45 as well :)